### DISCUSSION TITLE
Achievements Changes

### DISCUSSION THREAD

**Discussion by zain-asif-dev** [community/community#39709] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

There should be a page where you should be able to see all available achievement with how to earn them and progress.

**Comment by radar-blip** [comment, marked as answer]
[Achievement Badges](https://github.com/Schweinepriester/github-profile-achievements)

**Comment by jfullstackdev** [comment]
latest update, you can now pick what you want to hide, unlike in the past, it was all or nothing

go to Achievements Tab, click the `Achievements` word, it is a link

![image](https://user-images.githubusercontent.com/47092464/204711684-e7c5ebb6-57f9-4fbf-b8fe-96c2182f41bb.png)

or this URL, it's same thing 

https://github.com/[username]?tab=achievements

you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711801-dde96c4d-5133-4201-ad8a-9b23b55ed5d2.png)

then, click any badge, hover left then you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711940-2c5e8b4d-7580-4145-afc0-268f1739d8de.png)

then boom ... you now have the option to hide those badges that you want to hide
